week-1/errors/1.js

```diff
@@ -1,4 +1,5 @@
 // trying to create an age variable and then reassign the value by 1
 
-const age = 33;
+let age = 33;
 age = age + 1;
```

Comment:
very good with this reassignment! are you aware of this shorter way to write this line?